api/Document.json

```
@@ -8901,6 +8901,53 @@
             "standard_track": true,
             "deprecated": true
           }
+        },
+        "accepts_TrustedHTML": {
+          "__compat": {
+            "description": "Accepts `TrustedHTML` instances",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/write",
+            "spec_url": "https://html.spec.whatwg.org/multipage/dynamic-markup-insertion.html#dom-document-write",
+            "tags": [
+              "web-features:dom",
+              "web-features:trusted-types"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "83"
```

The mentioned WPT tests (https://wpt.live/trusted-types/Document-write.html) fail for me in Chrome 83 on BrowserStack Live, and only pass from Chrome 86.

```suggestion
                "version_added": "86"
```